gcp/cluster.tf

```diff
@@ -0,0 +1,103 @@
+resource "random_uuid" "cluster" {}
+
+locals {
+  timestamp = timestamp()
+}
+
+locals {
+  uuid = random_uuid.cluster.result
+}
+
+locals {
+  deployment_id = "${random_uuid.cluster.result}-${local.timestamp}"
```

Comment:
use `local.uuid` instead of the whole thing

Reply:
I just removed them haha.